bundles/aero.minova.rcp.preferencewindow/src/aero/minova/rcp/preferencewindow/control/TimeFormattingWidget.java

```diff
@@ -58,29 +61,36 @@ public Control build(final Composite parent) {
 
 		final Text text = new Text(cmp, SWT.BORDER);
 		addControl(text);
+		text.setMessage(DateTimeFormatterBuilder.getLocalizedDateTimePattern(null, FormatStyle.SHORT, Chronology.ofLocale(locale), locale));
 		text.setText(PreferenceWindow.getInstance().getValueFor(getCustomPropertyKey()).toString());
 		final GridData textGridData = new GridData(SWT.BEGINNING, SWT.CENTER, false, false);
 		textGridData.widthHint = 185;
 		text.setLayoutData(textGridData);
-		
+
 		Label example = new Label(cmp, SWT.NONE);
 		addControl(example);
 		final GridData exampleGridData = new GridData(SWT.FILL, SWT.CENTER, true, false);
 		example.setLayoutData(exampleGridData);
-		example.setText(getTimeStringFromPattern(text.getText()));;
-		
+		example.setText(getTimeStringFromPattern(text.getText()));
+
 		text.addListener(SWT.Modify, event -> {
 			PreferenceWindow.getInstance().setValue(getCustomPropertyKey(), text.getText());
 			example.setText(getTimeStringFromPattern(text.getText()));
 		});
 
 		return text;
 	}
-	
+
 	private String getTimeStringFromPattern(String pattern) {
 		try {
 			LocalTime lt = LocalTime.of(12, 35, 54);
-			DateTimeFormatter dtf = DateTimeFormatter.ofPattern(pattern, locale);
+			DateTimeFormatter dtf = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT).withLocale(locale);
+			if (pattern.contains("hh")) {
```

Comment:
Auch überprüfen, ob schon "a" eingegeben wurde, damit nicht zweimal am/pm dasteht

Reply:
Das a muss nun vom Benutzer selber eingegebene werden, da sowohl in der Message/Hint ein a bei jeweiligen Lokale steht und dieses im Tooltip beschrieben wird